src/constants.js

```diff
@@ -4,7 +4,7 @@ import { ACTOR_EVENT_NAMES } from '@apify/consts';
  * The default user agent used by `Apify.launchPuppeteer`.
  * Last updated on 2020-05-22.
  */
-export const DEFAULT_USER_AGENT = 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.67 Safari/537.36';
+export const DEFAULT_USER_AGENT = 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/101.0.4951.15 Safari/537.36';
```

Comment:
How big of a problem is that the OS will not match for Linux and Windows users @petrpatek?

Reply:
```suggestion
export const DEFAULT_USER_AGENT = 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/101.0.4951.54 Safari/537.36';
```

Reply:
It is essential if you don't alter the other navigator attributes such as `platform.` However, it is better than headless. If the dev wants a stealthy option, he should turn on the fingerprints.

Reply:
Changed the UA to linux based chrome, as that should be fine on platform, given we run things inside linux docker containers.